Campaign/management/commands/ProcessCampaignData.py

```diff
@@ -62,15 +62,12 @@ def _process_campaign_data(campaign, batch_user, campaign_type, max_count):
 
         try:
             task_cls.import_from_json(campaign, batch_user, batch_data, max_count)
-
         except Exception as e:
             raise CommandError(e)
-
         finally:
             batch_data.dataReady = True
             batch_data.activate()
             batch_data.save()
 
-    print('Campaign activated')
```

Comment:
Just checking, was this removed accidentally?

Reply:
Must've happened during the rebase. Latest commit adds this back.

Reply:
Thanks.